src/Form/input-classifiers.test.js

```diff
@@ -179,7 +179,19 @@ describe.each(testCases)('Test $html fields', (testCase) => {
                 inferredType: getInputSubtype(field),
                 manualScore: field.getAttribute('data-manual-scoring')
             }
-        })
+        }
+
+        const scores = manuallyScoredFields.map(getDetailsFromFailure)
+
+        const falseScores = automaticallyScoredFields
+            .filter(field =>
+                !manuallyScoredFields.includes(field) &&
+                field.getAttribute('data-ddg-inputtype') !== 'unknown' &&
+                field.tabIndex !== -1
```

Comment:
Here I am getting the fields that have been scored automatically but were not marked manually, meaning that the script is catching stuff that it shouldn't. I am excluding those that are marked `unknown` because we don't care about them, and those that are visually hidden, because we don't fill them so they are also harmless.